ubi9-openjdk-21.yaml

```diff
@@ -17,7 +17,7 @@ labels:
 - name: "maintainer"
   value: "Red Hat OpenJDK <openjdk@redhat.com>"
 - name: "com.redhat.component"
-  value: "openjdk-21-ubi9-container"
+  value: "openjdk-21-jlink-rhel9-container"
```

Comment:
The component needs to be `openjdk-21-jlink-tech-preview-ubi9-container` I think -- despite the image name having rhel9 in it. This change is part of #534 already, but I don't mind which PR goes in first.